recipes/gmap/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
     - single_quote_paths.patch
 
 build:
-  number: 0
+  number: 2
   binary_has_prefix_files:
```

Comment:
Removing the `binary_has_prefix_files` and `has_prefix_files` lists should fix things.

Reply:
> Removing the binary_has_prefix_files and has_prefix_files lists should fix things.

It didn't .. :( .. I will go for `sed`.

Reply:
As far as I can see, it did: https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/43996/workflows/165a2225-64f6-4590-bc3e-58db1057d1ef/jobs/148787 .
I'd revert https://github.com/bioconda/bioconda-recipes/pull/27820/commits/0fb47d643fb8f16b79f322d8717d4c24ecff0188 as this shouldn't be needed.

Reply:
You are right. The test for the perl script seems to be stalling because it waits for something from stdin. Also all the perl script seemingly do not support a help flag. Any idea how we could use the perl scripts?

Reply:
> Any idea how we could use the perl scripts?

Nope, I've never used them. But you could try with an empty stdin, e.g.,
```yaml
- ': | fa_coords'
```